src/WebJobs.Extensions.DurableTask/DurableClientAttribute.cs

```diff
@@ -9,12 +9,12 @@
 namespace Microsoft.Azure.WebJobs
 {
     /// <summary>
-    /// Attribute used to bind a function parameter to a <see cref="DurableOrchestrationClient"/> instance.
+    /// Attribute used to bind a function parameter to a <see cref="DurableClient"/> instance.
```

Comment:
Considering this documentation is public facing, but `DurableClient` is not, maybe we should reword this. I suggest:

Attribute used to bind a function parameter to an IDurableClient, IDurableOrchestrationClient, or an IDurableEntityClient.